tests/ClientTest.php

```diff
@@ -20,6 +20,20 @@ public function testConstructorDefaultUri()
         $this->assertEquals('mongodb://127.0.0.1/', (string) $client);
     }
 
+    /**
+     * @doesNotPerformAssertions
+     */
+    public function testConstructorAutoEncryptionOpts()
+    {
+        $autoEncryptionOpts = [
+            'keyVaultClient' => new Client(static::getUri()),
+            'keyVaultNamespace' => 'default.keys',
+            'kmsProviders' => ['aws' => (object) ['accessKeyId' => 'abc', 'secretAccessKey' => 'def']],
```

Comment:
Does the `kmsProviders.aws` option actually need to be cast to an object?

Reply:
No, that must have been left over from some testing. I've removed the cast.